packages/graph/src/gshape-prerendered-element.ts

```diff
@@ -27,16 +26,14 @@ import { GPreRenderedElement } from './gpre-rendered-element';
  * A popup model is rendered when hovering over a element and for many common use cases e.g rendering a tooltip
  * this model can be computed entirely on the server side.
  */
-export class GShapePreRenderedElement extends GPreRenderedElement implements GBoundsAware, GAlignable {
+export class GShapePreRenderedElement extends GPreRenderedElement implements GBoundsAware {
     static override builder(): GShapePreRenderedElementBuilder {
         return new GShapePreRenderedElementBuilder(GShapePreRenderedElement);
     }
 
+    position?: Point;
+    size?: Dimension;
     [GBoundsAware] = true;
-    [GAlignable] = true;
```

Comment:
ShaprePreRenderedElements are alignable in sprotty (https://github.com/eclipse/sprotty/blob/139fce21baab249b3ec9bc9e6d519558b876a8f9/packages/sprotty/src/lib/model.ts#L110) so we should also implement the interface